### ISSUE TITLE
Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by sabrehagen** [eslint/eslint#8427]
**Tell us about your environment**

* **ESLint Version:** 3.19.0
* **Node Version:** 7.7.2
* **npm Version:** 4.1.2

**What parser (default, Babel-ESLint, etc.) are you using?**

Default

**Please show your full configuration:**

```

{
  "extends": "airbnb",

  "env": {
    "node": true,
    "browser": true,
  },

  "globals": {
    "GLOBAL_ENV": true,
  },

  "rules": {
    "linebreak-style": 0,       // Both CRLF/LF are fine
    "semi": ["error", "never"], // Never allow semi colons
    "no-underscore-dangle": 0,  // Allow _ in var names such as _id
  }
}

```

**What did you do? Please include the actual source code causing the issue.**

I'm not exactly sure. I accidentally pressed a couple of keys at the same time, and this popup occurred. I doubt it was due to my key combination, although I cannot be sure. There should be enough information provided to diagnose. If not, please let me know what more you need.

The code below was the code in the currently active editor pane:

```js

import React, { Component, PropTypes } from 'react'

import UserNameSettings from 'stemn-shared/misc/UserSettings/UserNameSettings'
import UserEmailSettings from 'stemn-shared/misc/UserSettings/UserEmailSettings'
import InfoPanel from 'stemn-shared/misc/Panels/InfoPanel'

export default class SettingsAccount extends Component {
  render () {
    return (
      <div>
        <InfoPanel>
          <UserNameSettings entityModel={ 'auth.user' } />
        </InfoPanel>
        <InfoPanel>
          <UserEmailSettings />
        </InfoPanel>
      </div>
    )
  }

```

**What did you expect to happen?**

Not get this error.

**What actually happened? Please include the actual, raw output from ESLint.**

![screnshot](http://i.imgur.com/VCyzUWW.png)

```
Error: Cannot convert undefined or null to object Referenced from: /home/jackson/repositories/stemn-frontend/websiteNew/node_modules/eslint-config-airbnb-base/index.js Referenced from: airbnb Referenced from: /home/jackson/repositories/stemn-frontend/websiteNew/.eslintrc
TypeError: Cannot convert undefined or null to object
Referenced from: /home/jackson/repositories/stemn-frontend/websiteNew/node_modules/eslint-config-airbnb-base/index.js
Referenced from: airbnb
Referenced from: /home/jackson/repositories/stemn-frontend/websiteNew/.eslintrc
    at Function.from (native)
    at Object.<anonymous> (/home/jackson/repositories/stemn-frontend/websiteNew/node_modules/eslint/lib/rules/quotes.js:37:84)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.getHandler [as get] (/home/jackson/repositories/stemn-frontend/websiteNew/node_modules/eslint/lib/rules.js:72:16)
```

**Comment by not-an-aardvark** [comment]
The error is getting thrown from [here](https://github.com/eslint/eslint/blob/7489394b327144918bccee042676b4bbb0289a47/lib/rules/quotes.js#L37). It seems like `astUtils.LINEBREAKS` is undefined for some reason, but I can't imagine why that would be happening. It might be the same issue as https://github.com/eslint/eslint/issues/8059 (in which case reinstalling apparently fixes it).

**Comment by sabrehagen** [comment]
Thanks, a reinstall fixed this.
